src/index.ts

```diff
@@ -291,10 +291,13 @@ export class ExtensibilitySdk {
           'error'
         );
       } else if (ctx.loadTime > 2000) {
-        this.notify(
-          `Addon loading takes longer than 2 seconds. Load time:${ctx.loadTime}. Ready time: ${ctx.readyTime}. SessionId: ${sessionId}`,
-          'warning'
-        );
+        logger.current.log({
```

Comment:
We already log the messages in `handleLoadInfoMessage`. 
This was to notify the host app abount the long loading issue. If we remove it, how would the host app know about the loading times?

Reply:
The point is that we don't want to show this message to the user. 
<img width="766" alt="image" src="https://github.com/getoutreach/extensibility-sdk/assets/45597983/fcfa591f-7bb4-4261-817c-f9f3f1164cab">


Although I am one with you on that we should notify the owners on loading times or some issues, whenever they appear, but this doesn't seem to be the way as Taras wants it to be removed. As I mentioned in the description:
 _"It's probably worth to open a discussion on the notifications from SDK in the team once the whole team is together."_. 

This could be solved in a different ways: 

- Provide the developer some general performance information that could be visible for admins or developers only. Maybe even visible for only dev versions installed.
- Create a new type of notifications, that would only be visible to admins/developers.
- Anything else we come up with in such discussion.

But that's not the task of this ticket.

Reply:
Ok then, let's remove the notify here. There's one more `notify` a few lines above. Do we want to keep that one? I'd remove it.

Reply:
I was thinking about it, but considering it's an error and not a warning, it should be addressed by the developers. Therefor my initial idea was to keep it.